topic_monitor/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'topic_monitor'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
> this PR together with the one on ament_tool

ament tools isn't part of the debian package build. Can you explain why the PR to ament_tools would be needed to validate a debian build with this change?

With dh_python responsible for the direct packaging actions, we don't actually get to invoke setup.py directly. Instead, arguments are passed through the exported environment variables. [[1]](https://github.com/ros2/bloom/blob/b5d17cbbc9dbee3f00f5948ec47ef118fc9cec1e/bloom/generators/debian/generator.py#L539-L543).

The link above is to the ROS 2 bloom fork, the in-progress refactor uses separate templates for the different build types but interaction with setuptools via pybuild environment variables is still used. The ROS 2 bloom fork initially installed python packages into the default debian filesystem hierarchy, after some local testing, https://github.com/ros2/bloom/commit/9969bc18b0890324884e3763c864ace860f26945 and a typo fix for it, https://github.com/ros2/bloom/commit/22e272014ed96e720f74b6363914bb80dd54396d were added. In local testing the prefix alone wasn't sufficient to install python library files in the ROS installation prefix.

We added the install scripts together while I was in CA with https://github.com/ros2/bloom/commit/31fee87b20f6dfbbcf01aedc7776f8f50c64eab4 and its escape fixes https://github.com/ros2/bloom/commit/f221ee781754f661b03e9baec7cbc019d1cc0ce7 because the setting of `--prefix` alone wasn't enough to put the non-libexec binaries in the ROS installation prefix.


Unless the reason prefix was not being honored is identified and resolved for Bloom merging this as-is will require maintainers who want their scripts installed as libexec rather than bin will need to modify the bloom generated templates.